host-configs/ruby-toss_3_x86_64_ib-clang@10.0.0.cmake

```diff
@@ -69,17 +69,17 @@ set(ENABLE_GTEST_DEATH_TESTS ON CACHE BOOL "")
 
 # Root directory for generated TPLs
 
-set(TPL_ROOT "/usr/WS1/axom/libs/toss_3_x86_64_ib/2022_09_27_04_15_12/clang-10.0.0" CACHE PATH "")
+set(TPL_ROOT "/usr/WS1/axom/libs/toss_3_x86_64_ib/2022_12_08_21_44_21/clang-10.0.0" CACHE PATH "")
 
 set(CONDUIT_DIR "${TPL_ROOT}/conduit-0.8.3" CACHE PATH "")
 
 set(C2C_DIR "${TPL_ROOT}/c2c-1.3.0" CACHE PATH "")
 
-set(MFEM_DIR "${TPL_ROOT}/mfem-4.4.0" CACHE PATH "")
+set(MFEM_DIR "${TPL_ROOT}/mfem-4.5.0" CACHE PATH "")
 
 set(HDF5_DIR "${TPL_ROOT}/hdf5-1.8.22" CACHE PATH "")
 
-set(LUA_DIR "${TPL_ROOT}/lua-5.3.5" CACHE PATH "")
+set(LUA_DIR "/usr/WS1/axom/libs/toss_3_x86_64_ib/2022_12_08_21_44_21/clang-9.0.0/lua-5.4.4" CACHE PATH "")
```

Comment:
Note: These sorts of inconsistencies (e.g. clang@9 in a clang@10 host-config) should be fixed when we update uberenv